README.md

```diff
@@ -96,6 +96,12 @@ return static function (Config $config): void {
 
 # Available rules
 
+**Hint**: If you want to test how a Rule work, you can use the command like `phparkitect debug:expression <RuleName> <arguments>` to check which class satisfy the rule in your current folder.
+
+For example: `phparkitect debug:expression ResideInOneOfTheseNamespaces App`
+
```

Comment:
What do you think about adding a screenshot of the output is too much for this file?

Reply:
I think adding some screenshot to the README could be great, but this is not the most interesting part (the output is just a list of FQCN)...
Maybe we could create an issue to add a screenshot or a gif of phparkitect in action?

Reply:
yes, I think that we could add screenshots in another PR!